### PR TITLE
Changed Zomato folder

### PR DESCRIPTION
**Pull request by MohanaSrinivas**
## Description

> The changes made in this pull request should be briefly described.

#679 

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [ ] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots

Screenshot (113)

## Additional Notes/Comments

> Any additional remarks or suggestions concerning this pull request are welcome.

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: src/scrape_up/__init__.py
Comment (nikhil25803) [inline src/scrape_up/__init__.py:1]: What change is this? Do you think it makes any sense?

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/__init__.py
Comment (MohanaSrinivas) [reply src/scrape_up/__init__.py:1]: Yah I. Have added some code in zomato folder

Comment (nikhil25803) [reply src/scrape_up/__init__.py:1]: Check again in which folder you have added the code?

Comment (MohanaSrinivas) [reply src/scrape_up/__init__.py:1]: I have initiated pandas library to access zomato 
In __init__.py file

Comment (MohanaSrinivas) [reply src/scrape_up/__init__.py:1]: Screenshot 2023-08-07 095014
Screenshot 2023-08-07 095024
@nikhil25803

**Pull request closed without merging**